utils/config.go

```diff
@@ -563,27 +568,53 @@ func ValidateLdapFilter(cfg *model.Config) *model.AppError {
 	return nil
 }
 
-func ValidateLocales(cfg *model.Config) *model.AppError {
+func ValidateLocales(cfg *model.Config) (err *model.AppError) {
```

Comment:
Can you change this line back? We're not using named return values, so it isn't necessary